index.html

```diff
@@ -18193,6 +18193,7 @@ <h2 id="AudioWorkletGlobalScope">
 [Global=(Worklet, AudioWorklet), Exposed=AudioWorklet]
 interface AudioWorkletGlobalScope : WorkletGlobalScope {
     void registerProcessor (DOMString name, VoidFunction processorCtor);
+    readonly attribute double currentFrame;
     readonly attribute double currentTime;
```

Comment:
I am not sure about keeping both. However, we'll have to define `currentFrame` in the spec if we remove the current time here.

Reply:
JACK audio use integers for timing values, see: https://github.com/jackaudio/headers/blob/master/types.h

It could be helpful to check what other audio systems (like CoreAudio) do:

CoreAudio here: https://developer.apple.com/documentation/coreaudio/audiotimestamp

So "unsigned long long" possibly makes more sense ?

Reply:
Changed to unsigned long long.  We are now in an interesting situation where the current frame has higher resolution than current time.  But I'm not worried since even at 384 kHz, the max frame is like 1.5 million years from now.